images/macos/templates/macOS-11.pkr.hcl

```diff
@@ -0,0 +1,219 @@
+variable "xcode_install_password" { type = string }
+
+source "veertu-anka-vm-clone" "template" {
+  vm_name = "${var.vm_name}"
+  source_vm_name = "${var.source_vm_name}"
+  source_vm_tag = "${var.source_vm_tag}"
+  vcpu_count = "${var.vcpu_count}"
+  ram_size = "${var.ram_size}"
+  stop_vm =  "true"
+}
+
+build {  
+  sources = [        
+    "source.veertu-anka-vm-clone.template",
+  ]
+  provisioner "shell" {
+    inline = [
+      "mkdir ~/image-generation"
+    ]
+  }
+  provisioner "file" {
+    destination = "image-generation/"
+    sources     = [ "./provision/assets", "./tests", "./software-report", "./helpers" ]
+  }
+  provisioner "file" {
+    destination = "image-generation/add-certificate.swift"
+    source      = "./provision/configuration/add-certificate.swift"
+  }
+  provisioner "file" {
+    destination = ".bashrc"
+    source      = "./provision/configuration/environment/bashrc"
+  }
+  provisioner "file" {
+    destination = ".bash_profile"
+    source      = "./provision/configuration/environment/bashprofile"
+  }
+  provisioner "file" {
+    destination = "./"
+    source      = "./provision/utils"
+  }
+  provisioner "shell" {
+    inline = [
+      "mkdir ~/bootstrap"
+    ]
+  }
+  provisioner "file" {
+    destination = "bootstrap"
+    source      = "./provision/bootstrap-provisioner/"
+  }
+  provisioner "file" {
+    destination = "image-generation/toolset.json"
+    source      = "./toolsets/toolset-11.json"
+  }
+  provisioner "shell" {
+    scripts = [
+      "./provision/configuration/configure-tccdb-macos11.sh",
+      "./provision/configuration/add-network-interface-detection.sh",
+      "./provision/configuration/autologin.sh",
+      "./provision/configuration/disable-auto-updates.sh",
+      "./provision/configuration/screensaver-off.sh",
+      "./provision/configuration/ntpconf.sh",
+      "./provision/configuration/max-files.sh",
+      "./provision/configuration/shell-change.sh"
+    ]
+    environment_vars = [
+      "PASSWORD=${var.vm_password}",
+      "USERNAME=${var.vm_username}"
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; sudo {{ .Vars }} {{ .Path }}"
+  }
+  provisioner "shell" {
+    scripts = [
+      "./provision/configuration/preimagedata.sh",
+      "./provision/configuration/configure-ssh.sh",
+      "./provision/core/xcode-clt.sh",
+      "./provision/configuration/configure-machine.sh"
+    ]
+    environment_vars = [
+      "IMAGE_VERSION=${var.build_id}",
+      "IMAGE_OS=${var.image_os}"
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} {{ .Path }}"
+  }
+  provisioner "shell" {
+    script = "./provision/core/reboot.sh"
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; sudo {{ .Vars }} {{ .Path }}"
+    expect_disconnect = true
+  }
+  provisioner "shell" {
+    scripts = [
+      "./provision/core/homebrew.sh",
+      "./provision/core/powershell.sh",
```

Comment:
It looks like that the `pause_before` configuration is missed here. Is it expected?